scripts/sct_process_segmentation.py

```diff
@@ -589,13 +589,32 @@ def extract_centerline(fname_segmentation, remove_temp_files, verbose = 0, algo_
         file_results.write(str(int(i)) + ' ' + str(x_centerline_voxel[i - min_z_index]) + ' ' + str(y_centerline_voxel[i - min_z_index]) + '\n')
     file_results.close()
 
+    # create a .roi file
+    from datetime import datetime
+    date_now = datetime.now()
+    common_text_start = 'Begin Marker ROI\n  Build version="7.0_33"\n  Annotation=""\n  Colour=0\n  Image source="' + fname_segmentation + '"\n  Created  "' + date_now.strftime("%d %B %Y %H:%M:%S.%f %Z") + '" by Operator ID="SCT"\n'
```

Comment:
I will not delete it, I will update it. I would like to keep this option in `sct_process_segmentation`.

I will extract the code from optic, put it in a function and call it from `sct_process_segmentation`.